qiskit/tools/events/__init__.py

```diff
@@ -10,7 +10,24 @@
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
 
-"""Events
+"""
```

Comment:
It looks like this page is missing from the toctree, can you add it to the `qiskit/tools/__init__py`?